src/lib/rpc.ts

```diff
@@ -29,15 +35,19 @@ const fromAgoricNet = (str: string): Promise<MinimalNetworkConfig> => {
   return fetch(networkConfigUrl(netName)).then(res => res.json());
 };
 
-// XXX hard coded default to local
-let networkConfig: MinimalNetworkConfig = {
-  rpcAddrs: ['http://localhost:26657'],
-  chainName: 'agoric',
-};
-
 const makeAgoricNames = async (
   follow: (path: string) => Promise<ValueFollower<unknown>>,
 ) => {
+  const timeoutDurationMS = 10_000;
+  const timeout = setTimeout(
+    () =>
+      notifyError(
+        new Error(
+          'Connecting to RPC taking longer than expected. Check console for details.',
```

Comment:
"check console" is a good catch-all. though we should be careful that the console doesn't include red herrings

Reply:
True. Tested this with "Local" network when I didn't have a local chain running, this is what I see:

<img width="1196" alt="Screen Shot 2024-09-11 at 11 27 10 AM" src="https://github.com/user-attachments/assets/b74bce74-ff3e-4a00-8793-68cc0c4c600c">

We're not likely to see other errors when this happens, because the app won't try to load anything or connect to the wallet before rpcUtils is initialized.